### ISSUE TITLE
\pausenumbering\resumenumering not working in parallel text

### ISSUE THREAD

**Issue by floriandk** [maieul/ledmac#758]
# Version of reledmac / reledpar (if that is a development version, please precise the branch)
reledmac 2018/04/07 v2.26.7
reledpar 2018/04/07 v2.21.7

# If this is a newly introduced problem, please indicate the last working version of the package(s)
The oldest setup I could test is from TL 15 (reledmac 2016/03/23 v2.9.0 and reledpar 2016/03/23 v2.8.0) and displays the same behaviour as described below.

# Does the problem happen only when combining with some packages?
no

# Does the problem happens both in parallel typesetting and normal typesetting?
no, only with reledpar

# Description of the problem
when using `\pausenumbering`/ `\resumenumering` in parallel text linenumbers are not computed correctly/at all. The `reledmac reminder: 
 The number of the footnotes in this section has changed since the last run. ...` doesn't go away.

I'd like to use `\memorybreak`s as earlier: Though it might not be necessary any more, using it reduces compilation time considerably when chapters get very long. As parallel texts need many typesetting-runs to stabilise this quite important.

Apart from this there can also be other reasons to pause numbering, e.g. inserting shorter commenting sections between critical text. 

# Minimal Working Example

```` latex
\documentclass{book}


\usepackage[series={A},noeledsec,nofamiliar,noledgroup,draft]{reledmac}
\usepackage{reledpar}

\newcommand{\memorybreak}{\pausenumbering\resumenumbering}
%%no problem without pause-resumenumbering:
%\newcommand{\memorybreak}{}

%%same problem with endnotes:
%\let\Afootnote\Aendnote


\begin{document}

\lineation{page}
\firstlinenum*{1}
\linenumincrement*{1}


\begin{pages}
    \begin{Leftside}
        \beginnumbering
            \pstart
                Left side %
\edtext{paragraph}{%
	\Afootnote{test}}%
. Left side paragraph. Left side paragraph. Left side paragraph. Left side paragraph. Left side paragraph. Left side paragraph. Left side paragraph.
            \pend
            \memorybreak
            \pstart
                Other left side paragraph.  Other left side paragraph.  Other left side paragraph.  Other left side paragraph.  Other left side paragraph.  Other left side paragraph.  Other left side paragraph.  Other left side paragraph.  %
\edtext{Other}{%
	\Afootnote{test}}
 left side paragraph.  Other left side paragraph.
            \pend
        \endnumbering
    \end{Leftside}
    \begin{Rightside}
        \beginnumbering
            \pstart
                Right side paragraph. Right side %
\edtext{paragraph}{%
	\Afootnote{test}}.
. Right side paragraph. Right side paragraph.
            \pend
            \memorybreak
            \pstart
                Other right side paragraph. Other right %
\edtext{side}{%
	\Afootnote{test2}}
 paragraph. Other right side paragraph. Other right side paragraph. Other right side paragraph. Other right side paragraph. Other right side paragraph.
            \pend
        \endnumbering
    \end{Rightside}

\end{pages} 
\Pages

\doendnotes{A}

\end{document}
````

**Comment by maieul** [comment]
That does not make sense to make a \pausenumbering inside a a leftside/rightside environment for a need of alternating tests, because to be tyepset in parallel, texts need to be numbered section. For optimizing compilation time, I understand your problem. However, the problem seems to very depth in the code of reldedmac/reledpar, as it happens also with ledmac 0.7. As I have no time anymore for big improvement of ledmac/ledpar I won't take it for this low priority problem. But I keep your isse opened, maybe one day I would have time again

**Comment by floriandk** [comment]
I can very well imagine situations where I'd like to make an unnumbered insert into parallel texts, so this would be nice to have one day. But as I hadn't even realised until now that they don't work, it is obviously not often that one *really* needs to …

Regarding compilation-time I have full understanding that you cannot dedicate much time for minor problems. After all, I just need to make another cup of coffee while waiting ;)

**Comment by maieul** [comment]
for not numbered text inside parallel texts, it can't be possible
"directly" in any day, because of the way parallele typsetting work with
reledpar. You must play with \numberlinefalse....

**Comment by floriandk** [comment]
Thanks for the hint! `\numerlinefalse` would do precisely what I'd need. It just seems not to work in parallel text either. While my example works without `reledpar` the same text-snippet doesn't in the parallel part -- or am I using this in a wrong way?

```latex
\documentclass{book}


\usepackage[series={A},noeledsec,nofamiliar,noledgroup,draft]{reledmac}
\usepackage{reledpar}


\begin{document}

\lineation{page}
\firstlinenum*{1}
\linenumincrement*{1}

\beginnumbering
	       \pstart
                Left side %
\edtext{paragraph}{%
	\Afootnote{test}}%
. Left side paragraph. Left side paragraph. Left side paragraph. Left side paragraph. Left side paragraph. Left side paragraph. Left side paragraph.
            \pend
            \numberlinefalse
            \pstart
\textit{Add some text here that expplains something about the following passage, e.g. that it is only available in another ms because\footnote{regular footnote} of a lacuna in the main witness:            
}           \pend
			\numberlinetrue
            \pstart
                Other left side paragraph.  Other left side paragraph.  Other left side paragraph.  Other left side paragraph.  Other left side paragraph.  Other left side paragraph.  Other left side paragraph.  Other left side paragraph.  %
\edtext{Other}{%
	\Afootnote{test}}
 left side paragraph.  Other left side paragraph.
            \pend
\endnumbering



\begin{pages}
    \begin{Leftside}
        \beginnumbering
            \pstart
                Left side %
\edtext{paragraph}{%
	\Afootnote{test}}%
. Left side paragraph. Left side paragraph. Left side paragraph. Left side paragraph. Left side paragraph. Left side paragraph. Left side paragraph.
            \pend
            \numberlinefalse
            \pstart
\textit{Add some text here that expplains something about the following passage, e.g. that it is only available in another ms because\footnote{regular footnote} of a lacuna in the main witness:            
}           \pend
			\numberlinetrue
            \pstart
                Other left side paragraph.  Other left side paragraph.  Other left side paragraph.  Other left side paragraph.  Other left side paragraph.  Other left side paragraph.  Other left side paragraph.  Other left side paragraph.  %
\edtext{Other}{%
	\Afootnote{test}}
 left side paragraph.  Other left side paragraph.
            \pend
        \endnumbering
    \end{Leftside}
    \begin{Rightside}
        \beginnumbering
            \pstart
                Right side paragraph. Right side %
\edtext{paragraph}{%
	\Afootnote{test}}.
. Right side paragraph. Right side paragraph.
            \pend
            \numberlinefalse
            \pstart
            \pend
            \numberlinetrue
            \pstart
                Other right side paragraph. Other right %
\edtext{side}{%
	\Afootnote{test2}}
 paragraph. Other right side paragraph. Other right side paragraph. Other right side paragraph. Other right side paragraph. Other right side paragraph.
            \pend
        \endnumbering
    \end{Rightside}

\end{pages} 
\Pages

\end{document}
```

**Comment by maieul** [comment]
Stranges,
it seems me I did add  this feature for parallel typesetting. But
indeed, it not the case. It should be quite easy to add this feature,
but please open a new issue.

Best regards

Maïeul

**Comment by floriandk** [comment]
You definitely thought ablout it: it's even in the manual of `reledpar` ;)

new issue #760

Thanks again!

**Comment by maieul** [comment]
I close. In any way, I don't think split will optimize compilation time in // typesetting, as we will need more hack to allow that way.
